contracts/delegate/src/entry.rs

```diff
@@ -141,80 +141,81 @@ pub fn update_delegate_at_cell(
         return Err(Error::InputOutputAtAmountNotEqual);
     }
 
-    let (input_amount, input_delegate_at_data) =
+    let (input_delegate_at_amount, input_delegate_at_data) =
         get_delegate_at_data_by_lock_hash(&delegate_at_lock_hash, Source::Input)?;
-    let (output_amount, output_delegate_at_data) =
+    let (output_delegate_at_amount, output_delegate_at_data) =
         get_delegate_at_data_by_lock_hash(&delegate_at_lock_hash, Source::Output)?;
     if input_delegate_at_data.version() != output_delegate_at_data.version()
         || input_delegate_at_data.metadata_type_id() != output_delegate_at_data.metadata_type_id()
     {
         return Err(Error::UpdateDataError);
     }
+
+    let epoch = get_current_epoch(checkpoint_type_id)?;
     debug!(
-        "input_amount: {}, output_amount: {}",
-        input_amount, output_amount
+        "input_delegate_at_amount: {}, output_delegate_at_amount: {}, epoch: {}",
+        input_delegate_at_amount, output_delegate_at_amount, epoch
     );
 
-    let epoch = get_current_epoch(checkpoint_type_id)?;
-    // debug!("epoch: {}", epoch);
-    let mut at_change = 0i128;
+    let mut delegate_at_change = 0i128;
```

Comment:
The range of i128 is different from u128, but I don't have a better idea.

Reply:
i128 is also far from enough, it won't overflow